apps/frontend-menu-viewer/src/components/pages/Cart/OptionDropDown.tsx

```
@@ -0,0 +1,83 @@
+          />
+        </div>
+        <button onClick={() => setOpen((prev) => !prev)}>
+          {open ? <ChervonUpSvg /> : <ChervonDownSvg />}
+        </button>
+      </div>
+      {selectedOptionIds.size > 0 && (
+        <ul className="space-y-1 mt-5">
+          {group.items.map((item) =>
+            selectedOptionIds.has(item.id) ? (
+              <li className="text-mc-1 text-gray-800 flex items-center">
+                <CheckSvg />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)